packages/cli/test/composites.test.ts

```diff
@@ -56,17 +62,37 @@ describe('composites', () => {
       ).toBe(true)
     }, 60000)
 
-    test('composite creation succeeds', async () => {
+    test('composite creation succeeds but model is not deployed', async () => {
+      const ceramic = new CeramicClient()
+      const create = await execa('bin/run.js', [
+        'composite:create',
+        'test/mocks/composite.profile.post.schema',
+        `--did-private-key=${seed}`,
+      ])
+      const output = create.stdout.toString()
+      const def = JSON.parse(output) as EncodedCompositeDefinition
+      expect(def.version).toBe('1.1')
+      expect(Object.keys(def.version).length).not.toBe(0)
+      expect(def.aliases).toBeNull()
+      expect(def.views).toBeNull()
+      await expect(checkIfModelIndexed(ceramic, Object.keys(def.models)[0])).resolves.toBeFalsy()
```

Comment:
I'm surprised this test passes given that deploy is true by default  for `composite:create` in this version of the PR, and this test isn't explicitly passing `--no-deploy`.  What am I missing?